src/DraftJSEditor.jsx

```diff
@@ -13,6 +13,8 @@ import {
 
 import linkDecorator from './decorators/link';
 
+import '../lib/draftjs-editor.min.css';
```

Comment:
IIRC draftjs-editor assumes the app which depends on it will include the CSS itself if desired. This way we give end users the choice between importing/tweaking the Stylus styles or using the minified CSS directly.